public/Miscellaneous/printYoutube.js

```diff
@@ -1,35 +1,78 @@
-if (!window["youtubeIFrame"]) {
+﻿if (!window["youtubeIFrame"]) {
 	window["youtubeIframe"] = true;
 
 	window.addEventListener("load", () => {
 		
 		let candidates = $("iframe");
 		for (let i = 0; i < candidates.length; i++) {
 			let iframe = candidates[i];
-			if (iframe && iframe.src && !iframe.classList.contains("youtubeIFrame")) {
```

Comment:
I would recommending adding this back in since we are going to set this live and need to have this protection enabled.